docs/Data_Portal/Users_Guide/clinical_data_analysis.md

```diff
@@ -48,6 +48,33 @@ Note that the histogram plot applies to, and can be displayed for, both categori
 
 ### Survival Plot
 
+he survival analysis, which is seen in both the `Gene` and `Mutation` tabs, is used to analyze the occurrence of event data over time.  In the GDC, survival analysis is performed on the mortality of the cases. Thus, the values are retrieved from [GDC Data Dictionary](../../../Data_Dictionary) properties and a survival analysis requires the following fields:
```

Comment:
Maybe rephrase "Thus, the values are retrieved from properties and a survival analysis requires the following fields:" to "Survival analysis requires the following fields with values retrieved from the [GDC Data Dictionary](../../../Data_Dictionary):